src/databricks/labs/ucx/cli.py

```diff
@@ -113,6 +114,20 @@ def ensure_assessment_run():
         workspace_installer.validate_and_run("assessment")
 
 
+def revert_migrated_tables(schema: str, table: str):
+    ws = WorkspaceClient()
+    installation_manager = InstallationManager(ws)
+    installation = installation_manager.for_user(ws.current_user.me())
+    if not installation:
+        logger.error(CANT_FIND_UCX_MSG)
+        return None
+    warehouse_id = installation.config.warehouse_id
+    sql_backend = StatementExecutionBackend(ws, warehouse_id)
+    table_crawler = TablesCrawler(sql_backend, installation.config.inventory_database)
+    tm = TablesMigrate(ws=ws, backend=sql_backend, tc=table_crawler)
```

Comment:
Don't use keyed arguments when you can use positional arguments. This would make refactoring harder in the future

Reply:
Fixed

Reply:
This was not fixed. You still use named arguments

Reply:
I name the argument but not force naming it.

Reply:
Fixed